cmd/gen-release-notes/main.go

```diff
@@ -56,7 +57,8 @@ func main() {
 
 	if len(releaseNoteFiles) < 1 {
 		fmt.Fprintf(os.Stderr, "failed to find any release notes files.\n")
-		os.Exit(1)
+		//maps to EX_NOINPUT, but more importantly lets us differntiate between no files found and other errors
```

Comment:
Are you going to check the exit code for `66` somewhere else? Does it make sense to put it in a constant (or reusing an existing one)?

nit: typo in the comment.

Reply:
This is checked in the bash code in istio/test: https://github.com/istio/test-infra/pull/2886/files